src/RectTileLayer.ts

```diff
@@ -89,27 +89,32 @@ namespace pixi_tilemap {
             pb.push(textureIndex);
             pb.push(animCountX);
             pb.push(animCountY);
+            pb.push(animDuration);
 
             return this;
         }
 
         tileRotate(rotate: number) {
             const pb = this.pointsBuf;
-            pb[pb.length - 3] = rotate;
+            pb[pb.length - 4] = rotate;
         }
 
-        tileAnimX(offset: number, count: number) {
+        tileAnimX(offset: number, count: number, duration: number = 1) {
+            this.hasAnim = true;
             const pb = this.pointsBuf;
 
-            pb[pb.length - 5] = offset;
-            pb[pb.length - 2] = count;
+            pb[pb.length - 6] = offset;
+            pb[pb.length - 3] = count;
+            pb[pb.length - 1] = duration;
         }
 
-        tileAnimY(offset: number, count: number) {
+        tileAnimY(offset: number, count: number, duration: number = 1) {
+            this.hasAnim = true;
```

Comment:
I added this flag here (and in tileAnimX) for consistency with `addRect`, not sure if it was a bug or it is not required here.